common/scripted_effects/wc_sacrifice_scripted_effects.txt

```diff
@@ -121,59 +209,46 @@ caltulate_PREV_offmap_old_gods_currency_insanity_effect = {
 }
 
 add_offmap_old_gods_currency_insanity_after_calculation_effect = {
-	if = {
-		limit = { old_gods_offmap_decisions_enabled_trigger = yes }
-
+	PREV = {
```

Comment:
Why `PREV` is used here? From what I see it's needed only inside `PREV_sacrifice_THIS_to_old_god_effect`. Other places where `add_offmap_old_gods_currency_insanity_after_calculation_effect` is used don't have `PREV` scope at all.

Reply:
Fixed